src/main/java/seedu/address/model/person/PersonFieldsContainKeywordPredicate.java

```diff
@@ -39,7 +39,9 @@ public PersonFieldsContainKeywordPredicate(Map<String, String> categoryDescripti
      */
     @Override
     public boolean test(Person person) {
+        assert categoryDescriptionMap != null;
         boolean matchesCategoryDescription = categoryDescriptionMap.entrySet().stream().anyMatch(entry -> {
+            assert entry.getKey() != null;
```

Comment:
Nice assert. I forgot to do it, am not sure whether they will still allow me to do my asserts.